FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -524,7 +580,6 @@ float Maslow_::computeTL(float x, float y, float z){
     return sqrt(a*a+b*b+c*c) - (_beltEndExtension+_armLength);
 }
 
-
```

Comment:
i like this one

Reply:
Yeah, I was on the fence because like on the one hand having that extra space there leaves more room for activities, but on the other hand too much space can lead to it taking a long time to get from one part of the code to another. I see strong arguments for both sides on this one.